### PR TITLE
Distributed

### PR DESCRIPTION
**Pull request by hector0513**
image

### CODE REVIEW

File: zomes/holo_acorn/code/src/anchor/mod.rs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+extern crate serde_derive;
+extern crate serde_json;
+use hdk::entry_definition::ValidatingEntryType;
+use hdk::holochain_core_types::{
+    // agent::AgentId, dna::entry_types::Sharing, entry::Entry, link::LinkMatch,
+    dna::entry_types::Sharing,
+    entry::Entry,
+};
+
+pub fn init() {
```
Comment (Connoropolous) [inline zomes/holo_acorn/code/src/anchor/mod.rs]: I think `init` should return a `Result` type

File: zomes/holo_acorn/code/src/anchor/mod.rs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        "anchor".into(), // app entry type
+        // app entry value. We'll use the value to specify what this anchor is for
+        "goal_comments".into(),
+    );
+    let agents_anchor_entry = Entry::App(
+        "anchor".into(), // app entry type
+        // app entry value. We'll use the value to specify what this anchor is for
+        "agents".into(),
+    );
+    let _ = hdk::commit_entry(&goal_comment_anchor_entry);
```
Comment (Connoropolous) [inline zomes/holo_acorn/code/src/anchor/mod.rs]: These should then use `?` to return the Err as the Result if any of these calls fails for some reason

Comment (hector0513) [reply zomes/holo_acorn/code/src/anchor/mod.rs]: threw the wrong "?"

Comment (hector0513) [reply zomes/holo_acorn/code/src/anchor/mod.rs]: image
I'm making this mistake

Comment (Connoropolous) [reply zomes/holo_acorn/code/src/anchor/mod.rs]: Are you stuck on it?

Comment (hector0513) [reply zomes/holo_acorn/code/src/anchor/mod.rs]: yes, I am. can you help me? I wrote you on mattermost

**Pull request merged**